scss/footnote-legacy.scss

```
@@ -47,15 +47,7 @@
   overflow-y: scroll;
 
-  transform: scale(0.95);
-  opacity: 0;
-  transition: opacity 150ms ease, transform 150ms ease;
-
-  &.show {
-    opacity: 1;
-    transform: scale(1);
-    z-index: 100;
-  }
+  z-index: 100;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)